scripts/db_migrations/v4.10/addRiskNumbers.py

```diff
@@ -0,0 +1,37 @@
+import sys
+from pymongo import MongoClient
+
+if len(sys.argv) < 5:
+    print("Not enough arguments.")
+    print("addRiskNumbers.py <mongoURL> <mongoPort> <userName> <password>")
+    sys.exit(0)
+
+mongoURL = sys.argv[1]
+mongoPort = sys.argv[2]
+userName = sys.argv[3]
+password = sys.argv[4]
+
+connString = "mongodb://"+ userName + ":" + password +"@"+mongoURL + ":" + mongoPort + "/"
+
+##### Enable dry run to not commit to the database #####
+dryRun = True
+
+##### Connect to the Database #####
+db = MongoClient(connString)
+for database in db.database_names():
+    if database != "admin" and database != "local":
+        db = MongoClient(connString)[database]
+        print("--database:" + database)
+
+##### Get a model ID and find entries #####
+        for setting in db.settings.find(no_cursor_timeout=True):
+            modelId = setting.get("_id")
+            print("\t--model: " +  modelId)
+            riskNumber = 1
+            for risk in db[modelId + ".risks"].find():
```

Comment:
I think we want to sort this, whilst it's mostly true that they should be ordered by time naturally I don't think we can guarantee this.